src/Excel/Manager.php

```diff
@@ -354,14 +354,14 @@ public function getReader(File $file, array $options = [])
     public function getWriter(Spreadsheet $excel, File $file, array $options = [])
     {
         if (!$file->exists()) {
```

Comment:
Should this be renamed to getSize() as well?

Reply:
yeah, missed that.